src/pages/Main/Main.tsx

```diff
@@ -3,10 +3,19 @@ import './Main.css';
 import { Route } from 'react-router-dom';
 import { GetStarted } from 'pages/GetStarted/GetStarted';
 import { AddPhoto } from 'pages/AddPhoto/AddPhoto';
+import { CanvasDiv } from './CanvasDiv';
 
 export const Main: React.FC= () => {
+
+    function renderCanvas() {
+        return <CanvasDiv ></CanvasDiv>
+    }
+
     return <div className='mainContainer'>
         {/* Add route here to make component visible*/}
+        <React.Fragment>
```

Comment:
You can also add react fragments with <>... code...</>. No need to change it.

Reply:
good to know!